ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py

```diff
@@ -265,3 +265,45 @@ def test_hamming_window(
         beta=beta,
         dtype=dtype,
     )
+@handle_cmd_line_args
```

Comment:
We aren't using these decorators now, the refactored has changed someof the decorators as well as the code too. 
Kindly look into the document I've attached.
https://lets-unify.ai/ivy/deep_dive/ivy_tests.html#ivy-tests
You can refer to other functions as well, make sure your fork is up to date to see the latest changes in your local.